docs/ReleaseNewVersion.md

```diff
@@ -0,0 +1,39 @@
+# Releasing a new version
+
+This guide is to release a new version of the Code-checker plugin. Remember that when considering the version number to use, that
+ this project follows [Semantic Versioning](http://semver.org/), so bump the version number accordingly.
+
+Check the existing PRs and approve those that are applicable to this release.
+
+Create a standard PR with the version bump. Ensure the following have been updated:
+
+* The `CHANGE.md`, adding the PRs that have been approved and other existing changes.
+* The `version.php` file, at least the *version* and *release* values.
+
+Create a standard PR to get the version bump reviewed and incorporated upstream. Please **avoid merge commit** for the new
```

Comment:
Is this the same PR than the one before? (I think so). But reading it seems that there are 2 different PRs in the process. Maybe rewording it to make clear that it's the same one.

Reply:
It sounds confusing. Rewording...